wrapper/src/main/resources/aws_advanced_jdbc_wrapper_messages.properties

```diff
@@ -340,3 +338,16 @@ NodeResponseTimeMonitor.interruptedExceptionDuringMonitoring=Response time threa
 NodeResponseTimeMonitor.exceptionDuringMonitoringStop=Stopping thread after unhandled exception was thrown in Response time thread for node {0}.
 NodeResponseTimeMonitor.openingConnection=Opening a Response time connection to ''{0}''.
 NodeResponseTimeMonitor.openedConnection=Opened Response time connection: {0}.
+
+# Monitoring RDS HostList Provider
+ClusterTopologyMonitorImpl.startMonitoringThread=Start cluster topology monitoring thread for {0}.
+ClusterTopologyMonitorImpl.stopMonitoringThread=Stop cluster topology monitoring thread for {0}.
+ClusterTopologyMonitorImpl.exceptionDuringMonitoringStop=Stopping cluster topology monitoring after unhandled exception was thrown in monitoring thread for node {0}.
+ClusterTopologyMonitorImpl.invalidQuery=Error obtaining host list. Provided database might not be an Aurora Db cluster
```

Comment:
```suggestion
ClusterTopologyMonitorImpl.invalidQuery=An error occurred while attempting to obtain the topology because the topology query was invalid. Please ensure you are connecting to an Aurora Db cluster.
```